frontend/src/components/status/SystemCheck.jsx

```diff
@@ -47,25 +24,25 @@ class SystemCheck extends React.PureComponent {
     const { description, isPassing, currentSettingsDetails } = this.props;
 
     return (
-      <div style={styles.container}>
-        <div style={styles.icon}>
+      <tr>
+        <td>{description}</td>
+        <td>
           {isPassing && (
             <div style={styles.passing}>
-              <Glyphicon style={styles.glyphicon} glyph="ok-circle" />
-              <Label>{LOCALIZE.commons.passStatus}</Label>
-              <p style={styles.currentSpecs}>{currentSettingsDetails}</p>
+              <i style={styles.glyphicon} className="fas fa-check-circle" />
+              <span>{LOCALIZE.commons.passStatus}</span>
+              <span style={styles.currentSpecs}>{currentSettingsDetails}</span>
```

Comment:
Can we add a space between _**< span >**_ and _**{LOCALIZE.commons.passStatus}**_ and between _**< span style={styles.currentSpecs} >**_ and _**{currentSettingsDetails}**_, to have a little spaces between the check mark, the Pass word and the settings details like this:

![image](https://user-images.githubusercontent.com/23021242/52787775-e4648d00-302c-11e9-99a4-a5be234e142d.png)